Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM python:3.13.2-alpine
 
 WORKDIR /leqo-backend
 
+COPY ./.env /.env
+
```

Comment:
We should not need to copy the `.env` file as the environment can / should be set via docker (compose)